crates/hyperqueue/src/server/state.rs

```diff
@@ -20,6 +20,8 @@ use tako::{define_wrapped_type, TaskId};
 pub struct State {
     jobs: crate::Map<JobId, Job>,
     workers: crate::Map<WorkerId, Worker>,
+    // Here we store how many times was each Task present at worker crash
+    crashing_task_ids: crate::Map<TakoTaskId, u32>,
```

Comment:
My original idea was to put the counter into each task, but I like the idea of having a separate map as it would be mostly sparse in most cases (And in a case of mass fails, we probably consume more memory by storing error messages). 

I agree with @Kobzol that map should be stored in Job so we easily free memory related to Job once the job is removed.